packages/static-build/src/index.ts

```diff
@@ -647,23 +648,27 @@ export const build: BuildV2 = async ({
         };
       }
 
-      if (framework) {
```

Comment:
It seems like this code is being deleted?

Reply:
Huh, yeah it was. I restored it.